modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -95,7 +95,7 @@ public class RunQueryRoute extends RunsRoute {
 
 
 	private static final GalasaGson gson = new GalasaGson();
-	private final RasDetailsQueryParams rasDetailsQueryParams;
+	private boolean IS_METHOD_DETAILS_EXCLUDED = true;
```

Comment:
I think this field needs removing since excluding method details is done on a per-request basis, we can use a local variable inside the GET method instead